### PR TITLE
Annotate type of parameters and ports

### PR DESCRIPTION
**Pull request by accuminium**

### CODE REVIEW

File: src/doc/html/static/rustdoc.css [hunk 1]
```diff
@@ -568,6 +568,10 @@ h4 > code, h3 > code, .invisible > code {
 	flex-basis: 100%;
 }
 
+.type-annotation {
+	font-weight: 200;
+}
+
```
Comment (accuminium) [inline src/doc/html/static/rustdoc.css:574]: I was wondering if we should add an independent CSS file for customizations, so `rustdoc.css` can be easily updated from upstream.

Comment (zarubaf) [reply src/doc/html/static/rustdoc.css:574]: I think this is a good idea!

Comment (accuminium) [reply src/doc/html/static/rustdoc.css]: Done.

**Comment by zarubaf** [comment]
LGTM. Anything you want to add to this PR?

**Comment by accuminium** [comment]
No, ready from my side as well!

**Pull request merged**